wiki/quest-modding.md

```diff
@@ -1,14 +1,58 @@
 # Quest Modding
-_A quick summary of Quest Modding_
 
-## Quest Mods
-::: warning
-This page is a WIP and will be updated with time
-:::
+## Installation 
+
+Currently there are two working tools for installing custom songs and mods on your Quest:
+* [BeatSaberQuestInstaller (windows only GUI for installing BMBF)](https://github.com/NyanBlade/BeatSaberQuestInstaller/releases)
+* [BMBF installed via SideQuest](https://github.com/kihecido/BMBF/releases)
+
+For new users, BeatSaberQuestInstaller may be easier to use as it handles some of the more confusing parts of modding/updating a modded game, such as restoring scores and installing adb.
+
+### Installing with BeatSaberQuestInstaller
+
+Before starting the process go to the utilities tab and backup your scores, if you've previously modded Beat Saber you also need to go there to uninstall the game, make sure you backup **BEFORE** uninstalling.
+
+![BeatSaberQuestInstaller](./images/beginners-guide/qbsioptions.png)
```

Comment:
This is just a minor suggestion for future tutorial images.
The instructions say to backup before uninstalling, but the arrows don't indicated any order to click the buttons.
I think the order could be labelled clearly by numbering the arrows.
I also like to darken the unimportant parts of images like this, to make the arrows more obvious.
Here is a quick image of what I'm thinking that I put together in GIMP
![BeatSaberQuestInstaller](https://user-images.githubusercontent.com/27714637/72595871-a8325000-38bf-11ea-8329-d49665f8397d.png)

Reply:
thanks for the suggestion ill edit the images